datasets/superb/superb.py

```diff
@@ -148,6 +144,47 @@ def _info(self):
                 }
             ),
             supervised_keys=("file", "text"),
+            url="http://www.openslr.org/12",
+            data_url="http://www.openslr.org/resources/12/",
+            task_templates=[AutomaticSpeechRecognition(audio_file_path_column="file", transcription_column="text")],
+        ),
+        SuperbConfig(
+            name="sd",
+            description=textwrap.dedent(
+                """\
+            Speaker Diarization (SD) predicts `who is speaking when` for each timestamp, and multiple speakers can
+            speak simultaneously. The model has to encode rich speaker characteristics for each frame and should be
+            able to represent mixtures of signals. [LibriMix] is adopted where LibriSpeech
+            train-clean-100/dev-clean/test-clean are used to generate mixtures for training/validation/testing.
+            We focus on the two-speaker scenario as the first step. The time-coded speaker labels were generated using
+            alignments from Kaldi LibriSpeech ASR model. The evaluation metric is diarization error rate (DER)."""
+            ),
+            features=datasets.Features(
```

Comment:
do you happen to know if this schema allows one to easily compute the diarization error rate or is some additional preprocessing required?

if i understand correctly, for each frame, the fine-tuned models will predict logits per speaker so i'm wondering how we can connect this to the current schema?

the reason i'm asking is that ultimately i'd like to do something like the following during evaluation:

```
evaluation_dset = load_dataset("superb", "sd", split="test")
# dataset of predictions
submission_dset = load_dataset("json", data_files=["output-from-bulk-job.jsonl"])

metric = load_metric("der")
metric.compute(predictions=submission_dset["preds"], evaluation_dset[???])
```

but maybe we can deal with this once we start looking at the metric question :)

Reply:
@lewtun, as we agreed in our discussion in Slack with @Narsil and in our Google meeting, an additional step is required to transform current schema to the 2D array labels.

Below, I'm writing a summary of our discussion and the points we agreed on.